src/github/entities/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Base } from 'src/universal/base.entity';
+import { PullRequest } from 'src/webhook/entities/pull-request.entity';
+import { Repo } from 'src/webhook/entities/repository.entity';
+import { Column, Entity, OneToMany } from 'typeorm';
+import { UserInstallation } from './user-installation.entity';
+@Entity()
+export class User extends Base {
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  name: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  phone: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  email: string;
+
+  @Column({ type: 'varchar', nullable: true })
+  password: string;
+
+  @Column({ type: 'varchar', length: 20, unique: true })
+  githubUserId: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true, nullable: true })
+  githubUsername: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  githubAvatarUrl: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  githubAccessToken: string;
+
+  @Column({ type: 'varchar', nullable: true, length: 255 })
+  accessToken: string;
+  
+  @OneToMany(() => UserInstallation, (installation) => installation.user)
+  installations: UserInstallation[];
+
+  @OneToMany(() => Repo, (repo) => repo.user)
+  repo: Repo[];
```

Comment:
PullProofAI

It's recommended to add a constructor to initialize class properties in the `User` entity class.

```diff
@Entity()
+export class User extends Base {
+  constructor(data?: Partial<User>) {
+    super();
+    Object.assign(this, data);
+  }
```

#### Issue Found: Potential Security Vulnerability


<!-- This is an auto-generated comment by PullProofAI -->